cmd/gotk/get_helmrelease.go

```diff
@@ -62,26 +73,52 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	tableRowHeaders := []string{"Name", "Revision", "Ready", "Message", "Suspended"}
+	if allNamespaces {
+		tableRowHeaders = append(tableRowHeaders, "Namespace")
+	}
```

Comment:
```suggestion
	if allNamespaces {
		tableRowHeaders = append([]string{"Namespace"}, tableRowHeaders...)
	}
```

Had another look at `kubectl`, and it does add the namespace as the first column, this does just that. My apologies for not checking this beforehand.